From: Robert Schwebel <r.schwebel@pengutronix.de>
Subject: [patch] CHAR_MAX needs limits.h

Newer toolchains (tested with gcc-4.3.2, glibc-2.8) barf on undefined
constants, so we have to include limits.h here.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 src/candump.c     |    1 +
 src/canecho.c     |    1 +
 src/cansend.c     |    1 +
 src/cansequence.c |    1 +
 4 files changed, 4 insertions(+)

Index: canutils-1.1.2/src/candump.c
===================================================================
--- canutils-1.1.2.orig/src/candump.c
+++ canutils-1.1.2/src/candump.c
@@ -6,6 +6,7 @@
 #include <signal.h>
 #include <libgen.h>
 #include <getopt.h>
+#include <limits.h>
 
 #include <sys/types.h>
 #include <sys/socket.h>
Index: canutils-1.1.2/src/canecho.c
===================================================================
--- canutils-1.1.2.orig/src/canecho.c
+++ canutils-1.1.2/src/canecho.c
@@ -5,6 +5,7 @@
 #include <signal.h>
 #include <libgen.h>
 #include <getopt.h>
+#include <limits.h>
 
 #include <sys/types.h>
 #include <sys/socket.h>
Index: canutils-1.1.2/src/cansend.c
===================================================================
--- canutils-1.1.2.orig/src/cansend.c
+++ canutils-1.1.2/src/cansend.c
@@ -5,6 +5,7 @@
 #include <signal.h>
 #include <libgen.h>
 #include <getopt.h>
+#include <limits.h>
 
 #include <sys/types.h>
 #include <sys/socket.h>
Index: canutils-1.1.2/src/cansequence.c
===================================================================
--- canutils-1.1.2.orig/src/cansequence.c
+++ canutils-1.1.2/src/cansequence.c
@@ -5,6 +5,7 @@
 #include <signal.h>
 #include <libgen.h>
 #include <getopt.h>
+#include <limits.h>
 
 #include <sys/types.h>
 #include <sys/socket.h>
